dozer-sql/src/pipeline/builder.rs

```diff
@@ -284,25 +285,87 @@ fn select_to_pipeline(
 
     // Where clause
     if let Some(selection) = select.selection {
-        let selection = SelectionProcessorFactory::new(gen_selection_name.to_owned(), selection);
+        if let Expr::InSubquery {
+            expr,
+            subquery,
+            negated,
+        } = selection.clone()
+        {
+            // Subuery to join
```

Comment:
@mediuminvader Could you please take a look? Am I on the right track? Do you have any suggestions?

Reply:
oh, I guess I am totally wrong.
Things I need to do are:
1. Extend SelectionProcessor with a JoinOperator.
2. Support build the JoinOperator for SelectionProcessorFactory. 
3. Processing the data and using JoinOperator to check if it matches.

@mediuminvader @snork-alt Could you please take a look? Do you think this would work?

Reply:
That's an interesting approach, but I'm not getting point 1.
Can you ping me on discord? https://discord.gg/3tEvNd9y